src/sceneHandler/index.ts

```diff
@@ -196,6 +195,9 @@ export class SceneHandler {
     this.loadingScreen.initCloud();
     this.loading = true;
     document.getElementById("loading").style.zIndex = "3";
+    setTimeout(() => {
+      this.titleTextElement.style.zIndex =  "-1";
+    }, 800);
```

Comment:
What is the reason adding another setTimeout here? Is this gonna be called once?

Reply:
So that it will have a seamless effect after the clouds finish animating. Without it, the FTM title will just hide instantly.

Reply:
Then utilize CSS instead of creating another setTimeOut.

Reply:
Instead of zindex use property display:block/display:none, removing the element from DOM instead of just keeping it behind

Reply:
@bernhardccodev @ashwinnair-chimple pls check